packages/thirdweb/src/wallets/in-app/core/wallet/types.ts

```diff
@@ -41,6 +41,7 @@ export type InAppWalletCreationOptions =
         };
       };
       smartAccount?: SmartWalletOptions;
+      showPrivateKeyExport?: boolean;
```

Comment:
Wonder if we should make this true by default.

Also can you post a screenshot or better enable it in the playground?

Reply:
Agree. Changed it to `hidePrivateKeyExport` and added a demo video to the PR description